### PR TITLE
Completed javascript-forms exercise.

### PR DESCRIPTION
**Pull request by samlee0258**
Screenshot 2023-02-08 at 11 13 49 AM

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var $name = document.querySelector('#user-name');
+var $email = document.querySelector('#user-email');
+var $message = document.querySelector('#user-message');
+var $form = document.querySelector('#contact-form');
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var messageData = {
+    $name: $name.value,
```
Comment (robgmerrill) [inline javascript-forms/form-values.js:9]: For this exercise we want to access the values according to step 11 we want to use the `elements` property: `theForm.elements.name.value`

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Just a note on how this exercise expected you to get form values.

File: javascript-forms/form-values.js (hunk 1 above)
Comment (samlee0258) [reply javascript-forms/form-values.js]: Screenshot 2023-02-08 at 12 06 44 PM

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**